tests/e2e/pages/AnalyticsOverview.ts

```diff
@@ -1,24 +1,40 @@
+import { ElementHandle } from 'puppeteer';
 import { waitForElementByText } from '../utils/actions';
 import { Analytics } from './Analytics';
 
+type Section = {
+	title: string;
+	element: ElementHandle< Element >;
+};
+const isSection = ( item: Section | undefined ): item is Section => {
```

Comment:
Is this function just used to determine visibility?

`isVisible`?

Reply:
I use it further down in a `filter(isSection)`, this was the only way the type would correctly updated from: `Array<Section | undefined>` to `Array<Section>`. This way I don't have to specifically check if title is defined on a section within the tests.